child_compassion/mappings/childpool_search_mix_mapping.py

```diff
@@ -0,0 +1,33 @@
+# -*- encoding: utf-8 -*-
+##############################################################################
+#
+#    Copyright (C) 2016 Compassion CH (http://www.compassion.ch)
+#    Releasing children from poverty in Jesus' name
+#    @author: Michael Sandoz <michaelsandoz87@gmail.com>
+#
+#    The licence is in the file __openerp__.py
+#
+##############################################################################
+from openerp.addons.message_center_compassion.mappings.base_mapping import \
+    OnrampMapping
+
+
+class ChilpoolSearchMapping(OnrampMapping):
+    """
+    Childpool mapping for searching mix of available children from GMC.
+    """
+    ODOO_MODEL = 'compassion.childpool.search'
+    ACTION = "rich_mix"
+
+    CONNECT_MAPPING = {
+        'numberOfBeneficiaries': 'take',
+    }
+
+    FIELDS_TO_SUBMIT = {k: None for k in CONNECT_MAPPING.iterkeys()}
+
+    def _process_connect_data(self, connect_data):
```

Comment:
take is always set no ? So this method can be removed I think. Moreover, you could move that class in the same file as the search_mapping don't you think ?